gradle/libs.versions.toml

```diff
@@ -20,7 +20,7 @@ errorprone = "2.10.0"
 vertx = "4.2.2"
 
 versionsPlugin = "0.39.0"
-versionsFilterPlugin = "0.1.10"
+versionsFilterPlugin = "0.1.4"
```

Comment:
this needs to be either removed or replaced, otherwise it will be listed among the dependencies to upgrade

Reply:
i will do custom implementation and remove plugin